### ISSUE TITLE
Landmarks for non-active elements are present in the rotor for the redesigned homepage

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-cms#11827]
## Issue description
During screen reader testing, Tiffany identified two landmarks present in the rotor with a role=”region” that are not visible on the page. This means their presence is noted to screen reader users but not visible to sighted users, 

These labels might have inadvertently been copied from the existing/current homepage. 
- aria-label=”Maintenance banner” (_Per @laflannery, the empty maintenance banner is an existing backlog issue that occurs sitewide, not just on the homepage._ )
- aria-label=”Learn what the PACT Act means for your VA benefits” 

![image.png](https://images.zenhubusercontent.com/5d8a47393ac3440001d1a2ed/62b8fb54-94f3-4fad-a256-0472e7972821)

`<div aria-label="Maintenance banner" data-widget-type="maintenance-banner" role="region"></div>`

`<va-promo-banner aria-label="Learn what the PACT Act means for your VA benefits" role="region" data-template="components/banners.drupal.liquid" id="48052" href="/resources/the-pact-act-and-your-va-benefits" type="announcement" class="hydrated">Learn what the PACT Act means for your VA benefits</va-promo-banner>`

## Tasks
- Suggest removing or hiding those elements until content is made visible and active.  

Acceptance criteria
- Landmarks which are not visible and active are not present in the rotor or announced to screen readers

**Comment by mmiddaugh** [comment]
@humancompanion-usds - although this is outside our scope for the homepage, I thought I should make sure you are aware
